handbook/product/design/ways_of_working/index.md

```diff
@@ -0,0 +1,26 @@
+# Ways of working
+
+Sourcegraph’s product design team has a few different ways of working. Each designer may work in one or more ways at different times, depending on the company’s needs and priorities, the projects at hand, and the individual designer’s capacity and interests.
+
+Generally speaking, these can be described as working as embedded designers, and working as design consultants. We also work as a product design team, where our immediate peers are each other as designers.
+
+## Working as embedded designers
+
+We prefer to work as embedded designers whenever possible. In this way of working, a product designer is a member of a given cross-disciplinary team (like Batch changes, Search product, Extensibility, or Cloud SaaS). Each team has members from engineering, product management, and product design. This team collaborates closely day-by-day to move closer to achieving the team’s goals and long-term vision.
+
+Working as an embedded designer is the best way for us to collaborate end-to-end across our design process.
+When working as an embedded designer in a cross-disciplinary team, we join that team’s syncs, planning, retros, and other rituals, and also participate in defining and reaching the team’s OKRs.
```

Comment:
Should we add that designers should be cognizant of the number of team meetings on their calendar and judge for themselves what's necessary, as most designers are a part of multiple teams currently? As opposed to saying join all meetings for cross-disciplinary teams we are a part of?

I don't know if it will be problems in the future as our team expands, but it was something that Christina had pointed out to me in a 1-1.

Reply:
@jjinnii that sounds appropriate. You can use the icon that is the document with the + in it to suggest that change in a comment on this PR.